src/Symfony/Component/Console/Tests/DependencyInjection/AddConsoleCommandPassTest.php

```diff
@@ -76,6 +76,23 @@ public function testProcessRegisterLazyCommands()
         $this->assertSame(array('console.command.symfony_component_console_tests_dependencyinjection_mycommand' => false), $container->getParameter('console.command.ids'));
     }
 
+    /**
+     * @expectedException        \Symfony\Component\DependencyInjection\Exception\InvalidArgumentException
+     * @expectedExceptionMessage  Service "my-command" has multiple "console.command" tags with a different value for the "command" attribute, but the command must have only one name. Use the "alias" attribute instead.
+     */
+    public function testProcessThrowsIfCommandHasMultipleNames()
+    {
+        $container = new ContainerBuilder();
+        $container
+            ->register('my-command', MyCommand::class)
+            ->setPublic(false)
+            ->addTag('console.command', array('command' => 'my:command'))
+            ->addTag('console.command', array('command' => 'other:name'))
+        ;
```

Comment:
> Alternatively; `command: name, aliases: [name, nameN]`

Tag attributes cannot be arrays :) that's what I'm trying to mimic

> perhaps command is a bad name due term duplication with console.command, what about only name?

Not available :) `- { name: console.command, command: foobar }`

The `command` attribute is fine to me. What I don't like much is to call `setName()` on the command (mandatory) for the first `command` attribute found, and consider other as aliases. But if others think it's better, I'm ok for changing it